src/sql/workbench/contrib/executionPlan/browser/azdataGraphView.ts

```diff
@@ -85,11 +84,6 @@ export class AzdataGraphView {
 				}
 			}
 		});
-
-		// Focusing the parent container when we click the graph
-		this._diagram.graph.addListener('click', (sender, evt) => {
```

Comment:
Was this click handler moved into the azdataGraph library or is this just something that we no longer need?

Reply:
We are now focusing on the nodes itself instead of the parent container. Therefore, this event handler is not longer needed.